.github/workflows/goreleaser-fleet.yaml

```diff
@@ -76,7 +76,8 @@ jobs:
 
       - name: Run GoReleaser
         id: goreleaser
-        uses: goreleaser/goreleaser-action@f82d6c1c344bcacabba2c841718984797f664a6b
+        uses: goreleaser/goreleaser-action@90a3faa9d0182683851fbfa97ca1a2cb983bfca3 # v6.2.1
+
```

Comment:
Fixes the docker publish failures: https://github.com/fleetdm/fleet/actions/workflows/goreleaser-snapshot-fleet.yaml?query=branch%3Arc-minor-fleet-v4.64.0